src/site/artifacts/cargo_dist.rs

```diff
@@ -6,9 +6,18 @@ use axohtml::elements::{div, li, span};
 use axohtml::{html, text, unsafe_text};
 use cargo_dist_schema::{Artifact, ArtifactKind, DistManifest};
 
+pub fn get_os(name: &str) -> &str {
+    match name {
+        "x86_64-unknown-linux-gnu " => "linux",
+        "x86_64-apple-darwin " => "mac",
+        "x86_64-apple-silicon " => "mac",
+        "x86_64-pc-windows-msvc " => "windows",
+        &_ => "",
```

Comment:
we should put something for the base case or a comment that explains why this is not possible

Reply:
Done!